crates/sui-graphql-rpc/src/types/move_value.rs

```diff
@@ -75,6 +77,7 @@ pub(crate) struct MoveField {
     value: MoveData,
 }
 
+/// The value representation of a Move Object.
```

Comment:
This type is used for more things than just the value for a `Move object` (e.g. also events, dynamic field values, etc). How about:

```suggestion
/// An instance of a Move type.
```